<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:#DCDCDC;	
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color:#DCDCDC;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QPushButton" name="MemRecall">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #8A3939;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  rgb(208, 80, 80);	
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>MRC</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="4">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Pagul</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #FAEBD7;
	border: 1px solid gray;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="Clear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #8A3939;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  rgb(208, 80, 80);	
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="ClearOperations">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #C0C0C0;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #D3D3D3	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="MemAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #8A3939;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  rgb(208, 80, 80);	
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="MemSubtract">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #8A3939;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  rgb(208, 80, 80);	
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Pagul</family>
       </font>
      </property>
      <property name="text">
       <string>Operations</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="8">
     <widget class="QTextEdit" name="DisplayOperations">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="2" rowspan="2">
     <widget class="QPushButton" name="ChangeSign">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #A9A9A9;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #808080	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3" rowspan="2">
     <widget class="QPushButton" name="Button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #C0C0C0;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #D3D3D3	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4" rowspan="2">
     <widget class="QPushButton" name="Equals">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #A9A9A9;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #808080	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4" rowspan="2">
     <widget class="QPushButton" name="Button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #C0C0C0;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #D3D3D3	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3" rowspan="2">
     <widget class="QPushButton" name="Button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #C0C0C0;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #D3D3D3	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" rowspan="2">
     <widget class="QPushButton" name="Button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #C0C0C0;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #D3D3D3	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" rowspan="2">
     <widget class="QPushButton" name="Button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #C0C0C0;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #D3D3D3	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" rowspan="2">
     <widget class="QPushButton" name="Button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #C0C0C0;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #D3D3D3	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3" rowspan="2">
     <widget class="QPushButton" name="Button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #C0C0C0;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #D3D3D3	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" rowspan="2">
     <widget class="QPushButton" name="Button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #C0C0C0;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #D3D3D3	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" rowspan="2">
     <widget class="QPushButton" name="Button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #C0C0C0;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #D3D3D3	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" rowspan="2">
     <widget class="QPushButton" name="Button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #C0C0C0;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  #D3D3D3	;
	border: 1px solid gray;
 	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" rowspan="2">
     <widget class="QPushButton" name="Multiply">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #BD4141;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  rgb(208, 80, 80);	;
	border: 1px solid gray;
 	padding: 5px;
}
</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5" rowspan="2">
     <widget class="QPushButton" name="Subtract">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #BD4141;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  rgb(208, 80, 80);	;
	border: 1px solid gray;
 	padding: 5px;
}
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5" rowspan="2">
     <widget class="QPushButton" name="Add">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #BD4141;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  rgb(208, 80, 80);	;
	border: 1px solid gray;
 	padding: 5px;
}
</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5" rowspan="2">
     <widget class="QPushButton" name="Divide">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:  #BD4141;
	border: 1px solid gray;
 	padding: 5px;
}

QPushButton: pressed{
	background-color:  rgb(208, 80, 80);	;
	border: 1px solid gray;
 	padding: 5px;
}
</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
